src/main/scala/uk/gov/hmrc/play/frontend/bootstrap/FrontendGlobal.scala

```diff
@@ -81,7 +81,7 @@ abstract class DefaultFrontendGlobal
     super.onStart(app)
   }
 
-  def filters = if (enableSecurityHeaderFilter) frontendFilters ++ Seq(securityFilter) else frontendFilters
+  def filters = if (enableSecurityHeaderFilter) Seq(securityFilter) ++ frontendFilters  else frontendFilters
 
```

Comment:
@lohita-bandaru  The tests in FilterChainExceptionSpec are fine but they do not test this production code.  I think that at least you should add a test for FrontendGlobal.filters() which tests the actual code - as this is your above change could be reversed and your new test would still pass.